hosting/.gitignore

```diff
@@ -34,3 +34,5 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+yarn.lock
```

Comment:
We are currently using NPM for this project. Remove this.

Reply:
Done ✅